docs/rules/no-negated-condition.md

```diff
@@ -0,0 +1,65 @@
+# Disallow use of negated expressions in conditions (no-negated-condition)
+
+Checks against the use of a negated expression in an if condition when the else branch is empty or in a ternary operator.
```

Comment:
This description seems to not be inline with provided examples and code of the rule. It looks like if else branch is empty you exit the rule.